gnomad_mitochondria/pipeline/add_annotations.py

```diff
@@ -2064,6 +2135,9 @@ def main(args):
         help="Set to True to only include released gnomAD samples",
         action="store_true",
     )
+    parser.add_argument(
+        "--keep-all-samples", help="Set to True to keep all samples (will skip steps that filter samples because of contamination and/or mitochondrial copy number)", action="store_true",
+    )
```

Comment:
Run Black on file just to make sure it's all formatted 
```suggestion
    parser.add_argument(
        "--keep-all-samples", 
        help="Set to True to keep all samples (will skip steps that filter samples because of contamination and/or mitochondrial copy number)", 
        action="store_true",
    )
```